misc/build-checks/hlf-checker/CommandLine.cs

```diff
@@ -5,7 +5,7 @@
 using System.Linq;
 using System.Text.RegularExpressions;
 
-namespace HlfChecker
+namespace hlf_checker
```

Comment:
I would leave it as is, a Pascal Case is the correct (read "widely used") name convention for C# namespaces and classes.

Reply:
Yeah! Just wanted to experiment with the style of project names and directories which are more in line with everything else under FarManager\misc. It looks ugly indeed. Let me change it. It's still a draft, isn't it? :)

Reply:
So much better!